geodata/models/common.py

```diff
@@ -29,17 +26,21 @@ def save(self, *args, **kwargs):
         super(ModifiableEntry, self).save(*args, **kwargs)
 
 
-class SpatialEntry(ModifiableEntry):
-    """Common model to all geospatial data entries."""
+class SpatialEntry(models.Model):
+    """Common model to all geospatial data entries.
+
+    This is intended to be used in a mixin manner.
+
+    """
+
+    spatial_id = models.AutoField(primary_key=True)
```

Comment:
You'll need to change `def __str__(self):` to match, e.g., 
```
    def __str__(self):
        return '{} (type: {})'.format(self.spatial_id, type(self))
```

Or just remove the custom string method.